sprites/extra/extra-openttd-gui.pnml

```diff
@@ -319,3 +319,11 @@ replacenew ottd_gui175(OTTD_GUI, "", 175) {
 	[   44,   0,  20,  20,   0,   0, "sprites/png/gui/group_liveries.png"]         // boat group livery icon
 	[   66,   0,  20,  20,   0,   0, "sprites/png/gui/group_liveries.png"]         // aircraft group livery icon
 }
+replacenew ottd_gui179(OTTD_GUI, "", 179) { [   0,  0,  20,  20,   0,   0, "sprites/png/extra/extra-gui-build-tram.png"] } // global toolbar build tram
```

Comment:
Could split these over a few more lines, like the above?

Reply:
Follows apparent house style per https://github.com/OpenTTD/OpenGFX/blob/aeeb2f3c17d79b2cb567c5da9d3654843f9f56d8/sprites/extra/extra-openttd-gui.pnml#L88

I don't like it, but eh